PalasoUIWindowsForms/ImageToolbox/ArtOfReadingChooser.cs

```diff
@@ -218,8 +228,98 @@ private void ArtOfReadingChooser_Load(object sender, EventArgs e)
 		   // _searchTermsBox.Text = @"flower";
 			_searchButton_Click(this,null);
 #endif
+		}
 
-			_messageLabel.BackColor = Color.White;
+		private void SetMessageLabelText()
+		{
+			var msg = "In the box above, type what you are searching for, then press ENTER.".Localize("ImageToolbox.EnterSearchTerms");
+			// Allow for the old index that contained English and Indonesian together.
+			var searchLang = "English + Indonesian";
+			// If we have the new multilingual index, _searchLanguageMenu will be visible.  Its tooltip
+			// contains both the native name of the current search language + its English name in
+			// parentheses if its in a nonRoman script or otherwise thought to be unguessable by a
+			// literate speaker of an European language.  (The menu displays only the native name, and
+			// SearchLanguage stores only the ISO code.)
+			if (_searchLanguageMenu.Visible)
+				searchLang = _searchLanguageMenu.ToolTipText;
+			msg += Environment.NewLine + Environment.NewLine +
+					String.Format("The search box is currently set to {0}".Localize("ImageToolbox.SearchLanguage"), searchLang);
+			if (PlatformUtilities.Platform.IsWindows && !_searchLanguageMenu.Visible)
+			{
+				msg += Environment.NewLine + Environment.NewLine +
+						"Did you know that there is a new version of this collection which lets you search in Arabic, Bengali, Chinese, English, French, Indonesian, Hindi, Portuguese, Spanish, Thai, or Swahili?  It is free and available for downloading."
+							.Localize("ImageToolbox.NewMultilingual");
+				_downloadInstallerLink.Visible = true;
+				_downloadInstallerLink.BackColor = Color.White;
+				_downloadInstallerLink.Location = new Point(_downloadInstallerLink.Left, _messageLabel.Bottom + 4);
+			}
+			_messageLabel.Text = msg;
+		}
+
+		protected class LanguageChoice
+		{
+			static readonly List<string> idsOfRecognizableLanguages = new List<string> { "en", "fr", "es", "it", "tpi", "pt", "id" };
+			private readonly CultureInfo _info;
```

Comment:
I find it slightly surprising that this list is distinctly different (e.g., no Arabic) from the list we advertise AoR now supports.